services/user.py

```diff
@@ -0,0 +1,49 @@
+from db.models import User
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> None:
+    user = User.objects.get(id=user_id)
```

Comment:
The `update_user` function uses `User.objects.get(id=user_id)`, which will raise a `User.DoesNotExist` exception if no user with the given ID is found. Consider handling this exception or documenting that this function may raise an exception if the user is not found.